dev/tools/tokens/data/material-tokens.json

```diff
@@ -0,0 +1,1044 @@
+{
+  "version": "v0.72",
+  "date": "2021-12-16 00:27:25.239571",
+  "md.sys.color.light.on-tertiary": "md.ref.palette.tertiary100",
+  "md.sys.color.light.on-secondary-container": "md.ref.palette.secondary10",
+  "md.sys.color.light.on-secondary": "md.ref.palette.secondary100",
+  "md.sys.color.light.on-surface-variant": "md.ref.palette.neutral-variant30",
+  "md.sys.color.light.on-tertiary-container": "md.ref.palette.tertiary10",
+  "md.sys.color.light.secondary": "md.ref.palette.secondary40",
+  "md.sys.color.light.inverse-on-surface": "md.ref.palette.neutral95",
+  "md.sys.color.light.inverse-surface": "md.ref.palette.neutral20",
+  "md.sys.color.light.outline": "md.ref.palette.neutral-variant50",
+  "md.sys.color.light.on-primary": "md.ref.palette.primary100",
+  "md.sys.color.light.primary-container": "md.ref.palette.primary90",
+  "md.sys.color.light.tertiary": "md.ref.palette.tertiary40",
+  "md.sys.color.light.on-error": "md.ref.palette.error100",
+  "md.sys.color.light.shadow": "md.ref.palette.neutral0",
+  "md.sys.color.light.error-container": "md.ref.palette.error90",
+  "md.sys.color.light.surface-variant": "md.ref.palette.neutral-variant90",
+  "md.sys.color.light.background": "md.ref.palette.neutral99",
+  "md.sys.color.light.surface": "md.ref.palette.neutral99",
+  "md.sys.color.light.on-background": "md.ref.palette.neutral10",
+  "md.sys.color.light.tertiary-container": "md.ref.palette.tertiary90",
+  "md.sys.color.light.inverse-primary": "md.ref.palette.primary80",
+  "md.sys.color.light.error": "md.ref.palette.error40",
+  "md.sys.color.light.on-error-container": "md.ref.palette.error10",
+  "md.sys.color.light.on-surface": "md.ref.palette.neutral10",
+  "md.sys.color.light.primary": "md.ref.palette.primary40",
+  "md.sys.color.light.secondary-container": "md.ref.palette.secondary90",
+  "md.sys.color.light.on-primary-container": "md.ref.palette.primary10",
```

Comment:
I'm not sure what the official spec is, but I always thought it would be a nested json like
```json
"md.sys.color" : {
    "light" : {
         "on-tertiary" : "md.ref.palette.tertiary100",
         "on-secondary-container" : "md.ref.palette.secondary10",
    }
}
```
Without that it looks very verbose.

Reply:
ok, it seems there they do the same on js, why 😩